gulpfile.js

```diff
@@ -23,6 +23,17 @@ gulp.task('test', ['todo', 'lint'], function performTests() {
     .pipe(mocha({'reporter': 'spec', 'timeout': 100}));
 });
 
+gulp.task('test2', ['lint'], function performTests() {
```

Comment:
Im curious. Would adding the `process.exit` calls to the original test target have screwed up running gulp locally?